package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "devicon",
-  "version": "2.15.0",
+  "version": "2.15.1",
```

Comment:
This file shoulsn't be edited for icon requests. Please revert/reset the changes:)